### ISSUE TITLE
google-protobuf dependency doesn't work on Apple Silicon

### ISSUE THREAD

**Issue by ghost** [pganalyze/pg_query#204]
After upgrading to pg_query 2.0.2 I can't run rails server or console.

[server.log](https://github.com/pganalyze/pg_query/files/6246901/server.log)

**Comment by mikewindow** [comment]
https://github.com/protocolbuffers/protobuf/issues/8199 could be related

**Comment by mikewindow** [comment]
https://github.com/protocolbuffers/protobuf/issues/8372#issuecomment-802309499 is a workaround for this issue.

Add this to Gemfile:
```ruby
gem 'google-protobuf', github: 'bouk/protobuf', branch: 'ruby-build', submodules: true
```

**Comment by jirutka** [comment]
Why is this gem needed anyway? I’d really like to avoid this Google’s crap in the dependencies.

**Comment by lfittl** [comment]
This was resolved upstream in the protobuf 3.17.0 release, which the pg_query 2.1.0 release now depends on, thus closing this issue.
